de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5527010_sys_gh5374_Add_relation_SalesInvoice_DunningDocument.sql

```diff
@@ -0,0 +1,75 @@
+-- 2019-07-10T12:54:40.438
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Reference (AD_Client_ID,IsActive,Created,CreatedBy,IsOrderByValue,Updated,UpdatedBy,AD_Reference_ID,ValidationType,Name,AD_Org_ID,EntityType) VALUES (0,'Y',TO_TIMESTAMP('2019-07-10 12:54:40','YYYY-MM-DD HH24:MI:SS'),100,'N',TO_TIMESTAMP('2019-07-10 12:54:40','YYYY-MM-DD HH24:MI:SS'),100,541012,'T','C_DunningDoc_for_C_Invoice',0,'D')
+;
+
+-- 2019-07-10T12:54:40.451
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Reference_Trl (AD_Language,AD_Reference_ID, Help,Name,Description, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language, t.AD_Reference_ID, t.Help,t.Name,t.Description, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Reference t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N') AND t.AD_Reference_ID=541012 AND NOT EXISTS (SELECT 1 FROM AD_Reference_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Reference_ID=t.AD_Reference_ID)
+;
+
+-- 2019-07-10T12:57:42.581
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Ref_Table (AD_Reference_ID,AD_Key,WhereClause,AD_Client_ID,IsActive,Created,CreatedBy,Updated,IsValueDisplayed,AD_Window_ID,UpdatedBy,AD_Table_ID,AD_Org_ID,EntityType) VALUES (541012,547408,' EXISTS (     SELECT 1     from c_dunningdoc dd              INNER JOIN c_dunningdoc_line ddl on dd.c_dunningdoc_id = ddl.c_dunningdoc_id              INNER JOIN c_dunningdoc_line_source ddls on ddl.c_dunningdoc_line_id = ddls.c_dunningdoc_line_id              INNER JOIN c_dunning_candidate dc on ddls.c_dunning_candidate_id = dc.c_dunning_candidate_id              INNER JOIN c_invoice inv on dc.documentno = inv.documentno     where 1 = 1       AND inv.issotrx = ''Y''       AND C_dunningdoc.c_dunningdoc_id = dd.c_dunningdoc_id       AND inv.c_invoice_id = @C_Invoice_ID / -1@ )',0,'Y',TO_TIMESTAMP('2019-07-10 12:57:42','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2019-07-10 12:57:42','YYYY-MM-DD HH24:MI:SS'),'N',540155,100,540401,0,'D')
+;
+
+-- 2019-07-10T12:58:48.436
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_RelationType (Created,CreatedBy,IsActive,AD_Client_ID,Updated,UpdatedBy,AD_RelationType_ID,AD_Reference_Target_ID,AD_Org_ID,IsTableRecordIdTarget,IsDirected,AD_Reference_Source_ID,Name,EntityType) VALUES (TO_TIMESTAMP('2019-07-10 12:58:48','YYYY-MM-DD HH24:MI:SS'),100,'Y',0,TO_TIMESTAMP('2019-07-10 12:58:48','YYYY-MM-DD HH24:MI:SS'),100,540226,541012,0,'N','Y',540678,'C_Invoice -> C_DunningDoc','D')
```

Comment:
what i forgot: please set the `InternalName`column as well. Future-you will thank you when he can use it in his cypress test :-)

Reply:
fixed